explorer/websockethandlers.go

```diff
@@ -291,7 +293,9 @@ func (exp *explorerUI) RootWebsocket(w http.ResponseWriter, r *http.Request) {
 				case sigPingAndUserCount:
 					// ping and send user count
 					webData.Message = strconv.Itoa(exp.wsHub.NumClients())
-				case sigNewTx:
+				case sigNewTxs:
```

Comment:
`sigNewTx` is for internal signalling only now (e.g. `(*WebsocketHub).run` -> `(*explorerUI).RootWebsocket` handler in `explorer`, and `(*WebsocketHub).run` -> `(*PubSubHub).sendLoop` ).

`sigNewTxs` (`"newtxs") is for the message sent out to the remote client that is a slice of recent transactions.